firefox_media_tests/utils.py

```diff
@@ -24,20 +24,20 @@ def verbose_until(wait, target, condition):
         This is usually the input value provided to the `condition` used by
         `wait`. Ideally, `target` should implement `__str__`
     :param condition: callable function used by `wait.until()`
+    :param message: optional message to log when exception occurs
 
     :return: the result of `wait.until(condition)`
-    :raises: marionette.errors.TimeoutException
```

Comment:
@sydvicious So is that an r+? If so, could you update the flag on the bug, please? Thanks.